src/main/java/com/example/user_management/entity/User.java

```diff
@@ -0,0 +1,21 @@
+package com.example.user_management.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String username;
```

Comment:
If username is unique there should be unique constraint and other appropriate annotations

Reply:
Done